apps/notifications/src/modules/alert/providers/http-sdk.provider.ts

```diff
@@ -1,26 +1,26 @@
+import type { HttpClient } from "@akashnetwork/http-sdk";
 import { createHttpClient, DeploymentHttpService, LeaseHttpService } from "@akashnetwork/http-sdk";
-import type { Provider } from "@nestjs/common";
+import type { InjectionToken, Provider } from "@nestjs/common";
 import { ConfigService } from "@nestjs/config";
 
 import type { AlertConfig } from "@src/modules/alert/config";
 
+export const CHAIN_API_HTTP_CLIENT_TOKEN: InjectionToken<HttpClient> = Symbol("CHAIN_API_HTTP_CLIENT");
+
 export const HTTP_SDK_PROVIDERS: Provider[] = [
   {
-    provide: DeploymentHttpService,
+    provide: CHAIN_API_HTTP_CLIENT_TOKEN,
```

Comment:
non-blocking: why not using `HttpClient` as token?

Reply:
According to soli**D**, we must code everything around interfaces and `InjectionToken`s but this not always makes sense